rustls/src/common_state.rs

```diff
@@ -392,7 +392,7 @@ impl CommonState {
                         "QUIC uses TLS for the cryptographic handshake only"
                     );
                     let mut bytes = Vec::new();
-                    m.payload.encode(&mut bytes);
```

Comment:
Ditto, `get_encoding` candidate.

Reply:
can't do. `MessagePayload` has a custom `encode` method instead of implementing `Codec`.

Reply:
Fair enough!